server/consumer.go

```diff
@@ -1926,6 +1957,9 @@ func (o *consumer) loopAndGatherMsgs(qch chan struct{}) {
 			return
 		case <-mch:
 			// Messages are waiting.
+		case <-hbc:
+			hdr := []byte("NATS/1.0 200 Idle Heartbeat\r\n\r\n")
```

Comment:
200 seems weird, general optics is that it means things are done, no std code seems to fit the bill though

Reply:
Actually 202 Accepted seems better ```The request has been accepted for processing, but the processing has not been completed. The request might or might not be eventually acted upon, and may be disallowed when processing occurs.```

Reply:
I wanted to make sure folks did not think it was an error.. ;) Let me look for something better.

Reply:
Will go with 100 Continue.. Should I update description to Continue?